src/app/DashboardViewFactory.java

```diff
@@ -35,16 +42,18 @@ private DashboardViewFactory() {}
 
     public static DashboardView create(DashboardViewModel dashboardViewModel,
                                        DeleteProjectViewModel deleteProjectViewModel,
+                                       RefactorProjectViewModel refactorProjectViewModel,
                                        ViewManagerModel viewManagerModel,
                                        AddProjectDataAccessInterface addProjectDataAccessInterface,
                                        NotificationUsersDataAccessInterface notificationUsersDataAccessInterface,
-                                       DeleteProjectDataAccessInterface deleteProjectDataAccessInterface) {
+                                       DeleteProjectDataAccessInterface deleteProjectDataAccessInterface,RefactorProjectDataAccessInterface refactorProjectDataAccessInterface) {
```

Comment:
reformat this so the RefactorProjectDataAccessInterface is on a separate line, there is a lot going on in this method, it's easier if we keep it organized.